src/scenes/home/index.jsx

```diff
@@ -13,9 +13,15 @@ import {
 import { Alert } from '@material-ui/lab'
 import { Delete } from '@material-ui/icons'
 
+import React from 'react'
+import Modal from 'react-modal'
+
 import { api } from "../../data"
 import { validateInput } from "../../utils/validators"
 
+//This is needed so screen readers don't see main content when modal is opened.
```

Comment:
Hey that's a cool feature!

Though unfortunately it's breaking our unit tests right now. The issue is that we're rendering the `<App />` component in one of our tests, and the component itself has no concept of the "#root" element in the html file which the react app hooks into.

The fix for this is to put this line into the component
`if (process.env.NODE_ENV !== 'test') Modal.setAppElement('#root')`
so that this can be skipped during tests.

source: https://github.com/reactjs/react-modal/issues/632#issuecomment-378755186